horizon/scripts/onms-sentinel-init.sh

```diff
@@ -48,13 +45,10 @@ OPENNMS_DATABASE_CONNECTION_MAXPOOL=${OPENNMS_DATABASE_CONNECTION_MAXPOOL-50}
 NUM_LISTENER_THREADS=${NUM_LISTENER_THREADS-6}
 KAFKA_SASL_MECHANISM=${KAFKA_SASL_MECHANISM-PLAIN}
 KAFKA_SECURITY_PROTOCOL=${KAFKA_SECURITY_PROTOCOL-SASL_PLAINTEXT}
-ELASTICSEARCH_INDEX_STRATEGY_FLOWS=${ELASTICSEARCH_INDEX_STRATEGY_FLOWS-daily}
-ELASTICSEARCH_REPLICATION_FACTOR=${ELASTICSEARCH_REPLICATION_FACTOR-2}
-ELASTICSEARCH_NUM_SHARDS=${ELASTICSEARCH_NUM_SHARDS-6}
 
 # Wait for Dependencies
-if [[ -v ELASTICSEARCH_SERVER ]]; then
-  wait_for ${ELASTICSEARCH_SERVER}
+if [[ -v ELASTICSEARCH_HOSTS ]]; then
+  wait_for ${ELASTICSEARCH_HOSTS}
```

Comment:
I'm not sure how well the `wait_for` function works for multiple hosts; we may need to improve it.

Reply:
I know this is broken and I'm looking at what I can do to handle that.